framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -3931,19 +3931,24 @@ private void checkPreAndPostConditions() {
           overriderType, overriddenType, superCPostFalse2, subCPostFalse2, postfalsemsg);
     }
 
+    /**
+     * Checks member reference receivers.
```

Comment:
What in particular is this checking for?

Reply:
I haven't read the method to understand what it is checking.  I only edited the method to us the new util method.  I can just remove the Javadoc if you think that's better.

Reply:
I think that's probably better.  Thanks.  You can create the special SKIP-REQUIRE-JAVADOC file.